### DISCUSSION TITLE
Group Assignment #5 Rough Draft

### DISCUSSION THREAD

**Discussion by albertkun** [albertkun/21S-ASIAAM-191A#150] [Assignments]
#### Due 5/27
For [Group Assignment #5](
https://github.com/albertkun/21S-ASIAAM-191A/blob/main/Week_8/Materials/groupassignment.md), create a rough draft of your website and be prepared to give a short 2-3 minute presentation with your group members. 

### Submission
Post the link to your GitHub repository and website below.

ex.
**Group Repo:** https://github.com/akochaphum/my-awesome-group
**Group website:** https://akochaphum.github.io/my-awesome-group/index.html

**Comment by rachan2023** [comment]
repo: https://github.com/rachan2023/21S-191A-Against-Asian-Hate
website: https://rachan2023.github.io/21S-191A-Against-Asian-Hate/Final%20project/

**Comment by Wny-Duong** [comment]
repo: https://github.com/Wny-Duong/WebDev-GIS-Health-Equity
website: https://wny-duong.github.io/WebDev-GIS-Health-Equity/leaflet-storymaps-with-google-sheets-main/

(Screwed up trying to submit this, also storymap is a pain to use)

**Comment by lvalles10** [comment]
Repo: https://github.com/lvalles10/Reimagine-6th
Website: https://lvalles10.github.io/Reimagine-6th/index.html

**Comment by KenNL42** [comment]
Repo: https://github.com/KenNL42/ASIAAM-191A-Final-Project
Website: https://kennl42.github.io/ASIAAM-191A-Final-Project/index.html

**Comment by epham36** [comment]
Repo: https://github.com/jason-muljadi/SEAing-Color
Website: https://jason-muljadi.github.io/SEAing-Color/

**Comment by skarakas1** [comment]
repo: https://github.com/skarakas1/Tenants-Rights-Mapping-Project
website: https://skarakas1.github.io/Tenants-Rights-Mapping-Project/landing_page.html

**Comment by annyrviloria** [comment]
repo: https://github.com/annyrviloria/21S-ASIAAM-191A-GroupProject/tree/main/Rough_Draft
website: https://annyrviloria.github.io/21S-ASIAAM-191A-GroupProject/Rough_Draft/
